pyproject.toml

```diff
@@ -191,7 +191,13 @@ sdist.include = ["fast_pauli/__version__.py"]
 write_to = "fast_pauli/__version__.py"
 
 [tool.cibuildwheel]
-build = ["cp310-manylinux_x86_64", "cp311-manylinux_x86_64", "cp312-manylinux_x86_64", "cp313-manylinux_x86_64"]
-build-frontend = "build"
+skip = "pp*"
 test-requires = "pytest"
 test-command = "pytest -s -vv --import-mode importlib {project}/tests/fast_pauli"
+
+[tool.cibuildwheel.linux]
+build = ["cp3*_x86_64"]
+skip = ["*musllinux*"]
```

Comment:
Out of curiosity @sebastroy were you running into problems with `musllinux` based systems?

Reply:
YEah it looks like from c librairies are missing. I plan to try those again later